tests/functional/storageclass/test_cross_sc_clone_snap_restore.py

```diff
@@ -128,15 +128,21 @@ def test_cross_class_same_pool_clone_snap_restore(
         restore_pvc_obj2.delete()
 
     @pytest.mark.parametrize(
-        argnames="interface_type",
+        argnames=["interface_type", "replica1", "replica2"],
         argvalues=[
-            pytest.param(*[constants.CEPHBLOCKPOOL]),
-            pytest.param(*[constants.CEPHFILESYSTEM]),
+            pytest.param(*[constants.CEPHBLOCKPOOL], "", ""),
+            pytest.param(*[constants.CEPHFILESYSTEM], "", ""),
+            pytest.param(*[constants.CEPHBLOCKPOOL], 3, 2),
+            pytest.param(*[constants.CEPHFILESYSTEM], 3, 2),
+            pytest.param(*[constants.CEPHBLOCKPOOL], 2, 3),
+            pytest.param(*[constants.CEPHFILESYSTEM], 2, 3),
```

Comment:
can you add some comments for this value explanation, its confusing since its mapping like replica1=3 & replica1=2

Reply:
Done, arguments were given meaningful names and docstring edited.